Source/Core/DolphinWX/VideoConfigDiag.cpp

```diff
@@ -137,6 +137,7 @@ static wxString use_ffv1_desc = wxTRANSLATE("Encode frame dumps using the FFV1 c
 #endif
 static wxString free_look_desc = wxTRANSLATE("This feature allows you to change the game's camera.\nMove the mouse while holding the right mouse button to pan and while holding the middle button to move.\nHold SHIFT and press one of the WASD keys to move the camera by a certain step distance (SHIFT+2 to move faster and SHIFT+1 to move slower). Press SHIFT+R to reset the camera and SHIFT+F to reset the speed.\n\nIf unsure, leave this unchecked.");
 static wxString crop_desc = wxTRANSLATE("Crop the picture from its native aspect ratio to 4:3 or 16:9.\n\nIf unsure, leave this unchecked.");
+static wxString square_pixels_desc = wxTRANSLATE("Force Dolphin to output square pixels instead of the correct PAL or NTSC aspect ratios.\n\n If unsure, leave this unchecked.");
```

Comment:
Really the answer should be: "To prove someone wrong in an argument"

Some games might look subjectively better with this option, or closer to what the developers/artists intended, but the developers/artists would have seen the correct aspect ratio when they were testing on the real hardware, so they approved the game for shipping with the correct aspect ratios. 

In other cases the game might internally render with an incorrect aspect ratio but the art will compensate for the wrongness. 

Really you could write an entire easy in that box about if this option should be enabled or not, but I think it might be best to say something like: "If you know what you are doing."